subprojects/testing-jvm/src/integTest/resources/org/gradle/testing/junit/JUnitJdkNavigationTest/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.apache.fexli:org.osgi.foundation:1.2.0'
```

Comment:
Odd, good catch. I've been running `./gradlew test` -- is that not picking up these tests?